app/models/site.rb

```diff
@@ -5,7 +5,7 @@ class Site
   include ActiveModel::Serialization
 
   # Number of categories preloaded when lazy_load_categories is enabled
-  LAZY_LOAD_CATEGORIES_LIMIT = 50
+  LAZY_LOAD_CATEGORIES_LIMIT = 10
```

Comment:
Is there a reason we are reducing the limit here?

Reply:
To make other category scaling bugs more obvious, but I can move to a different PR.